File: js/src/maplestory/api/kms/dto/guild/guild.ts
import * as base from '../../../common/dto/guild/guild';
import { GuildBody } from '../../response/guild/guildBody';

/**
 * 길드 식별자 정보
 */
export class GuildDto extends base.GuildDto {
  /**
   * 길드 식별자
   */
  public override oguildId: string;

  constructor(obj: GuildBody) {
    super();

    const { oguild_id } = obj;

    this.oguildId = oguild_id;
  }
}

File: csharp/MapleStory.OpenAPI/Src/TMS/Dto/Guild/Guild.cs
using Newtonsoft.Json;
using Base = MapleStory.OpenAPI.Common.DTO;

namespace MapleStory.OpenAPI.TMS.DTO
{
    /// <summary>
    /// 公會識別碼資訊
    /// </summary>
    public class GuildDTO : Base.GuildDTO
    {
        /// <summary>
        /// 公會識別碼
        /// </summary>
        [JsonProperty("oguild_id")]
        public override string OGuildId { get; set; }
    }
}
